sound/soc/sof/sof-audio.h

```diff
@@ -344,6 +344,7 @@ struct snd_sof_pcm {
 	struct list_head list;	/* list in sdev pcm list */
 	struct snd_pcm_hw_params params[2];
 	bool prepared[2]; /* PCM_PARAMS set successfully */
+	bool stream_stopped[2]; /* only used if (!pcm_ops->platform_stop_during_hw_free) */
```

Comment:
based on the code it is only used **if (pcm_ops->platform_stop_during_hw_free)**

Reply:
Can we rename this flag to `stream_restart_needed` or `trigger_stop_pending` or something else?
We really set this flag when we _don't_ stop the stream and we use it as an indication that the stream was _not_ stopped to stop it in prepare (when hw_params() is skipped).